package.json

```diff
@@ -31,6 +31,7 @@
         "start:default": "nx serve demo -c def",
         "start:shadow": "nx serve demo -c shadow",
         "start:host": "nx serve demo --open --host 0.0.0.0 --disable-host-check",
+        "start:ssl": "./scripts/install-ssl.sh && nx serve demo --ssl=true --sslCert=localhost.pem --sslKey=localhost-key.pem",
```

Comment:
We already have the such command:

https://github.com/taiga-family/taiga-ui/blob/3c1ebcd5d07d249e314ebf1f938ac186996c52fa/projects/demo/project.json#L165-L169

Reply:
Remove duplicated script, added alias to root `package.json`